apps/emqx/src/emqx_persistent_session_ds/emqx_persistent_session_ds_inflight.erl

```diff
@@ -0,0 +1,357 @@
+%%--------------------------------------------------------------------
```

Comment:
Note: this file was renamed from `_ds_buffer` to inflight, but Git treats this as change. There are not many actual changes in this file.